proposal/ankit7540/proposal.rst

```diff
@@ -0,0 +1,154 @@
+-----------------------
+Proposal for Confocal
+-----------------------
+
+A package for modelling light intensity profile in a confocal microscope using Huygens-Fresnel principle.
+
+https://github.com/ankit7540/Confocal
+
+Problem to solve
+################
+
+Confocal microscopes achieve looking at a specific plane in some object which can be a liquid or a living cell. This 2D plane has a finite thickness
+which governs the volume observed by the microscope. In spectroscopic applications, the **volume of the medium observed** by the confocal
+microscope is important (as in my reasearch). Determination of this volume by theoretical modelling is the aim of the present work. Applications of this volume are numerous, such as, estimation of intrinsic property of molecules to give some signal, which can 
+be determined if the concentration in a liquid and the confocal volume is known.
+
+General scheme of a confocal microscope is given below. (Image from Olympus microscopy)
+
+.. image:: cfm.png
+    :width: 220px
+    :align: center
+    :height: 225px
+    :alt: alternate text
+
+Observed volume at the focal plane is a 3D bi-conical section. (Image from ThorLabs)
+
+.. image:: cfm2.jpg
+    :width: 180px
+    :align: center
+    :height: 200px
+    :alt: alternate text
+
+**Field** : Optics, Spectroscopy
+
+**Theory** : Huygens-Fresnel principle
+
+**Algorithm** : 3D integrals of complex valued functions (for numerical integration, Gauss-Legendre Quadrature or similar implementations) For a full mathematical description refer to the paper mentioned in the reference.
+
+
+Perspective users
+################
+
+Researchers using confocal microscopes
+
+System architecture
+################
+
+General idea:
+
+ 1. There are 5-8 number of parameters required to model a microscope. Plan to used some defaults and some user input values to set up the calculation. ( For example, mesh/grid density defined using step size for setting up matrix representing 3D space, focal length of lenses, wavelength of light, and integral limits) 
```

Comment:
Do you need to optimize for the parameters?  If you do, you can probably also describe how the optimization may work.